skbase/tests/test_base.py

```diff
@@ -56,8 +92,73 @@ class FixtureClassChild(FixtureClassParent):
 FIXTURE_OBJECT_TAGS = {"A": 42424241, "B": 3, "C": 1234, 3: "E"}
 
 
+class InvalidInitSignatureTester(BaseObject):
+    def __init__(self, a, *args):
+        pass
+
+
+class Example(BaseObject):
```

Comment:
to avoid side effects - since these are invoked by reference, and you could in theory write class variables in one test - would that be avoided by using proper pytest fixtures?

Seems complicated and low risk though.

Reply:
Going the fixture route, shouldn't be too bad.

Reply:
Have now used fixtures.